tcpdump.c

```diff
@@ -2020,6 +2020,18 @@ main(int argc, char **argv)
 		if (dlt == DLT_LINUX_SLL2)
 			fprintf(stderr, "Warning: interface names might be incorrect\n");
 #endif
+	} else if ( dflag && yflag_dlt ) {
+		/*
+		 * We're dumping the compiled code with a
+		 * user-specified DLT.
+		 */
+		/*
+		 * If no snapshot length was specified, or a length of 0 was
+		 * specified, default to 256KB.
+		 */
+		if (ndo->ndo_snaplen == 0)
+			ndo->ndo_snaplen = 262144;
```

Comment:
s/262144/MAXIMUM_SNAPLEN/ ?
(`netdissect.h:322:#define MAXIMUM_SNAPLEN        262144`)

Reply:
Sure, I just copied this code from `open_interface()`.